### PR TITLE
Adds article about Mirage serializers

### PR DESCRIPTION
**Pull request by jorgelainfiesta**
## Motivation

Min did an extensive research on Mirage serializers 

## Approach

Min provided me with a draft, then I poked his brain for days until this piece finally its final, extensive, shape.

[Rendered article](https://deploy-preview-96--frontside.netlify.app/blog/2020-08-31-mirage-serializers/)

### CODE REVIEW

File: src/pages/blog/2020-08-31-mirage-serializers.md [hunk 1]
```diff
@@ -0,0 +1,488 @@
...
+      "author_id": "...",
+      "post_blog_comments_ids": [...]
+    }
+  ]
+}
+```
+
+### Field names format: under_scored
+
+The Active Model Serializer changes the caps to be under_scored no matter how the original model name was configured. As you can see in the example, the attribute names that were inconsistently cased are all under_scored now; for instance, `blogPost_title` becomes `blog_post_title`.
```
Comment (minkimcello) [inline src/pages/blog/2020-08-31-mirage-serializers.md:244]: https://deploy-preview-96--frontside.netlify.app/blog/2020-08-31-mirage-serializers/#field-names-format-under_scored

i think we need to escape the underscore

Comment (minkimcello) [reply src/pages/blog/2020-08-31-mirage-serializers.md:244]: i was reading it going "whoa, this is dramatic" then thought maybe it wasn't intentional

**Review by minkimcello — CHANGES_REQUESTED** [review]

**Comment by minkimcello** [comment]
I found two small errors ☝️.  Otherwise it looks great! 🙏

File: src/pages/blog/2020-08-31-mirage-serializers.md (hunk 1 above)
Comment (jorgelainfiesta) [reply src/pages/blog/2020-08-31-mirage-serializers.md:244]: haha good catch! fixing it now

**Review by minkimcello — APPROVED** [review]

**Review by cherewaty — APPROVED** [review]
👏 👏 👏

**Comment by cowboyd** [comment]
I love it! I have only two suggestions

1. we link to the simulation article at the end as "related", but I think we could make an actual hyperlink to it when we say "we prefer to think of this as simulation"
2. Can we give mirage graphql honorable mention at the end as something that let's you not have to worry about serializers ever?

**Comment by jorgelainfiesta** [comment]
Wow I didn't even know Mirage GraphQL existed... 🤯

**Pull request merged**